pkg/kubelet/eviction/mock_threshold_notifier.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eviction
+
+import (
+	mock "github.com/stretchr/testify/mock"
```

Comment:
Here's your problem. Mocks need to either: be in a separate package (`package eviction_test`), separate directory (`testing` subdir & package), or file with the `_test.go` suffix.

Reply:
yeah, moved to a _test.go file